src/utils/MarkdownHighlighter.js

```diff
@@ -0,0 +1,18 @@
+import lazyhljs from './LazyLoader/hljs';
+import '../components/css/highlight.css';
```

Comment:
Better, than it the Markdown component, but I would push it even further down the stack, right down to `src/third-party/highlight.js`